superset/assets/src/SqlLab/components/SqlEditorLeftBar.jsx

```diff
@@ -41,12 +41,10 @@ class SqlEditorLeftBar extends React.PureComponent {
   onDatabaseChange(db, force) {
     const val = db ? db.value : null;
     this.setState({ schemaOptions: [] });
+    this.setState({ tableOptions: [] });
```

Comment:
A single `setState` call with `({ schemaOptions: [], tableOptions: [] })` would be better as it would trigger a single render call instead of 2.